src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,65 @@
+import { useEffect, useState } from 'react';
+import { People } from '../components/People';
+import { Loader } from '../components/Loader';
+import { Person } from '../types';
+
+export const PeoplePage = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState<string | null>(null);
+  const [people, setPeople] = useState<Person[] | null>(null);
+
+  useEffect(() => {
+    const fetchPeople = async () => {
+      try {
+        setIsLoading(true);
+        setError(null);
+
+        const response = await fetch(
+          'https://mate-academy.github.io/react_people-table/api/people.json',
+        );
+
+        if (!response.ok) {
+          throw new Error('Failed to fetch people');
+        }
+
+        const data = await response.json();
+
+        setTimeout(() => {
+          setPeople(data);
+        }, 500);
+      } catch (err) {
+        setError('Something went wrong');
+      } finally {
+        setIsLoading(false);
+      }
+    };
+
+    fetchPeople();
+  }, []);
+
+  return (
+    <div className="container">
+      <h1 className="title">People Page</h1>
+
+      <div className="block">
+        <div className="box table-container">
+          {isLoading && <Loader />}
+
+          {error && (
+            <p data-cy="peopleLoadingError" className="has-text-danger">
+              {error}
+            </p>
+          )}
+
+          {!isLoading && !error && people !== null && people.length === 0 && (
```

Comment:
The condition `people !== null && people.length === 0` checks if the people array is empty. Ensure that this condition is necessary and correctly reflects the intended logic.